big.js

```diff
@@ -1067,11 +1069,11 @@
     P.toFixed = function (dp) {
         var str,
             x = this,
-            neg = TO_EXP_NEG,
-            pos = TO_EXP_POS;
+            neg = Big.TO_EXP_NEG,
```

Comment:
As before, `Big = x.constructor,` would need to be included before this, otherwise `Big` will refer always to the original `Big` constructor.